### PR TITLE
refactor(ios): disable navigation gesture with js

### PR DESCRIPTION
**Pull request by CatsJuice**
Do not toggle navigation gesture native to avoid unexpected behavior. close AF-1958, AF-1797

### CODE REVIEW

**Comment by CatsJuice** [comment]
* **#9287** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9287?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9287?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `canary`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/toeverything?ref=merge-queue-instructions-in-stack-comment&prId=9287).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: packages/backend/server/src/schema.gql [hunk 1]
```diff
@@ -11,11 +11,6 @@ type BlobNotFoundDataType {
   spaceId: String!
 }
 
-enum ChatHistoryOrder {
```
Comment (darkskygit) [inline packages/backend/server/src/schema.gql]: should not commit this change

Comment (CatsJuice) [reply packages/backend/server/src/schema.gql]: Why this file always appear! How to avoid this?

Comment (darkskygit) [reply packages/backend/server/src/schema.gql]: backend module will enable based on env or config, in this case, define `COPILOT_OPENAI_API_KEY=1` and `COPILOT_FAL_API_KEY=1` to enable copilot module

**Review by Brooooooklyn — APPROVED** [review]

**Pull request merged**